app/models/app.rb

```diff
@@ -5,5 +5,6 @@ class App < ActiveRecord::Base
   validates :subdomain, uniqueness: true
   validates :subdomain, :intercom_id, format: { with: /\A[a-zA-Z0-9]*\z/, message: ": only letters and/or numbers are allowed"}
```

Comment:
@Aslonski should we just replace what's on this line with your change? i.e. `/\A[a-zA-Z0-9]*\z/` becomes `/\A[a-z0-9]+\z/`?